jenkins/ec2_bootstrap.sh

```diff
@@ -423,6 +423,44 @@ prepare_cowbuilder() {
     /bin/bash ./bootstrap.sh
 }
 
+install_datadog_agent() {
+    # Install the Datadog agent. The agent requires an API key which is expected
+    # to be present in the shell environment as DATADOG_AGENT_API_KEY. If the
+    # value is not present then inform stdout but do not fail the run.
+    if [[ -z "${DATADOG_AGENT_API_KEY}" ]]; then
+	echo "Optional envvar DATADOG_AGENT_API_KEY does not exist."
+    else
+	# PC1 == Puppet Collection 1 == most stable for this platform
+	PUPPET_REPO_URI='https://apt.puppetlabs.com/puppetlabs-release-pc1-wheezy.deb'
+	(cd /tmp && \
+	  wget --no-check-certificate "${PUPPET_REPO_URI}" && \
+	  dpkg -i puppetlabs*.deb)
+
+	set -e
+	
+	apt-get update
```

Comment:
Since you haven't added new lines to the apt sources.list. you don't need to run 'apt-get update' again. Please don't.

Yes, this could break if we change the order in which the functions are run. Let's not do that.